### ISSUE TITLE
Improve string formatting and context

### ISSUE THREAD

**Issue by bitigchi** [calamares/calamares#2204]
While updating the Turkish translations, I've noticed that there are multiple areas that the messages can be improved. Since Calamares is a crucial application for having a bootable system, I think this subject is rather important.

I will list them as concise as possible:

- Strings with variable placeholders need comments with either variable explanations, or an example that shows a full message. This will greatly reduce errors, and help translators, especially with languages with different word ordering.
- ALL the progress strings that are displayed during the install process should use the format/grammar "Deleting files…"; with an ellipsis indicating that there is a process going on. Currently most of these strings use something like "Set this and that." with a period at the end. Usually this format indicates documentation text, not progress. Some of the progress strings just use the format "This Process" with title case, with no comment whatsoever. This is really confusing for the translator, and causes ambiguity problems with languages with different 
- All sentences that end with a period should indicate informative text (emphasising above), not action or progress
- Buttons should use "Title Case" (KDE HIG)
- Titles should use "Title Case" (KDE HIG)
- Tooltips should use "Sentence case" (KDE HIG)
- Labels should use "Sentence case" (KDE HIG)
- Placeholder text should use "Sentence case" (KDE HIG)
- Most importantly, all of these strings should have the necessary comments to indicate their type, so that the translator can know the relevant context

I believe, adhering to the points will greatly improve the interface of Calamares, and make it fit in more with KDE software. Thanks for reading.

**Comment by adriaandegroot** [comment]
This would work a lot better with specific pull requests; they don't have to address everything, but they can improve things step-by-step. For instance, the debug window (if you run `calamares -d` there is a hardly-visible button in the lower-right to open the debug window), you can see on-screen that it has text "Debug information" in English. You can look for that string in Transifex and find pointers to the source location, or search for it in the Calamares source-code with `grep` or similar tools.

```
src/calamares/DebugWindow.cpp
164:    CALAMARES_RETRANSLATE( m_ui->retranslateUi( this ); setWindowTitle( tr( "Debug information" ) ); );
```

There it is! You could file a PR fixing the title-case and, optionally, providing more context.

**Comment by bitigchi** [comment]
I plan to spend some time on this in the near future.

**Comment by adriaandegroot** [comment]
Fixed with all of the PRs, ending in #2260
